core/src/main/java/io/questdb/griffin/SqlKeywords.java

```diff
@@ -637,6 +637,47 @@ public static boolean isHourKeyword(CharSequence tok) {
                 && (tok.charAt(i)) == 'r';
     }
 
+    public static boolean isMinuteKeyword(CharSequence tok) {
+        if (tok.length() != 6) {
+            return false;
+        }
+
+        int i = 0;
+        return (tok.charAt(i++)) == 'm'
```

Comment:
We have shortcuts in SAMPLE BY syntax for minute, hour, day etc, I'd prefer it to be supported here too